packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -282,6 +282,7 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
       version: {
         type: "TIMESTAMP WITH TIME ZONE",
         notNull: true,
+        default: "now()",
```

Comment:
Just flagging that this would result in all the entries within a bulk insertion having the same timestamp (transaction time). We could use `clock_timestamp()` to get the actual time instead

Reply:
I wasn't able to use `clock_timestamp()` because of our migration script. Turned out that I had to wrap it in `pgm.func()`: 58bc68b8164910c2a8575842a495d73de231d413

Reply:
> Just flagging that this would result in all the entries within a bulk insertion having the same timestamp (transaction time)

Isn't this fine?

Reply:
If you create an entity and update that in the same transaction, you would get a duplicated version (we had that problem in tests).

Reply:
In some ways that seems fine (I appreciate it's not okay if we're using it as a unique primary key). I'd consider the operation to happen at the point the transaction committed.. And if I was bulk inserting a bunch of things at once I'd kind of want to have them appear at once..

Creating and updating something in the same transaction is the thing that seems more strange.

Anyway, not asking for changes.